exceptiongen/__init__.py,sha256=RbpIe-JPFs4BM9MkCu456QJYTHdCL-9CRTN8uscsRt8,445
exceptiongenerator-2.1.dist-info/DESCRIPTION.rst,sha256=OCTuuN6LcWulhHS3d5rfjdsQtW22n7HENFRh6jC6ego,10
exceptiongenerator-2.1.dist-info/METADATA,sha256=ykWSlYQLs9O9Mgt9Sb5j9AojP07NPQGwcTkdk7YHckw,315
exceptiongenerator-2.1.dist-info/RECORD,,
exceptiongenerator-2.1.dist-info/WHEEL,sha256=BTHHj6QhsHpWhRUrxioIvCAgjx81u-a8gkPTO6bi_A0,116
exceptiongenerator-2.1.dist-info/metadata.json,sha256=JmYgeM_0RuU__41HVwo13gSob5z_-KApvZj6c9wYvMQ,496
exceptiongenerator-2.1.dist-info/top_level.txt,sha256=vjzNUzV67RifbiU_t2Fbj2Si8c0By8uyEsC1SNZABBU,13
exceptiongenerator-2.1.dist-info/zip-safe,sha256=frcCV1k9oG9oKj3dpUqdJg1PxRT2RSN_XKdLCPjaYaY,2
